cogs5e/sheetManager.py

```diff
@@ -502,6 +502,61 @@ async def _confirm_overwrite(self, ctx, _id):
                 f"If you only wanted to update your character, run `{ctx.prefix}update` instead.")
         return True
 
+    @commands.command(name='import')
+    @commands.max_concurrency(1, BucketType.user)
+    async def import_sheet(self, ctx, url: str, *args):
+        """
+        Loads a character sheet in one of the accepted formats:
+            [Dicecloud](https://dicecloud.com/)
+            [GSheet v2.1](http://gsheet2.avrae.io) (auto)
+            [GSheet v1.4](http://gsheet.avrae.io) (manual)
+            [D&D Beyond](https://www.dndbeyond.com/)
+        
+        __Valid Arguments__
+        `-nocc` - Do not automatically create custom counters for class resources and features.
+
+        __Sheet-specific Notes__
+        D&D Beyond:
+            Private sheets can be imported if you have linked your DDB and Discord accounts.  Otherwise, the sheet needs to be publicly shared.
+        Gsheet:
+            The sheet must be shared with Avrae for this to work.
+            Avrae's google account is `avrae-320@avrae-bot.iam.gserviceaccount.com`.
+
+        Dicecloud:
+            Share your character with `avrae` on Dicecloud (edit permissions) for live updates.
+        """
+        loading = await ctx.send('Loading character data to Avrae...')
+        prefix = ''
+        parser = None
```

Comment:
These 3 lines can be removed (the first sends a duplicate message, and the latter two are guaranteed to be set in one of the if bodies)

Reply:
Although the message needs to be moved before `extract_gsheet_id_from_url` in the google case in this case